content/foundation/faq.md

```diff
@@ -162,3 +162,13 @@ If you have sent us mail because you think you have traced spam to a system that
 
 - How to: Look up the owner of an IP address<small>(such as 10.0.35.147): [https://ws.arin.net/whois/](https://ws.arin.net/whois/). ARIN (Registered Internet Numbers) database (tells you who owns an IP address, such as 10.0.35.147, or which other database to query if the address is assigned outside the USA).
 
+## Marketing
+
+### Can I refer to an individual, organization or community as “founding” an ASF project? 
+Many projects enter the Apache Incubator with a recognized individual founder, group of founders, or corporate entity where the project originated and/or which was responsible for its development/administration. 
+
+The ASF is vendor neutral: all ASF projects are overseen by their respective Project Management Committees (PMCs), who guide the project's day-to-day operations. Once a project comes to the ASF, its development and oversight is the responsibility of the project’s PMC and community. 
+
+There are no "owners" of any Apache project. Individuals and companies who founded projects that are now under the auspices of the ASF may be referred to as "original creator of Apache Project Name" or "original developer of Apache Project Name." Examples: 
+* **Individual’s title.** "Individual’s Name, Apache Member (or Committer) and original developer of Apache Project Name." 
+* **Company whose founders or employees are the original creators of an Apache project.** "Company Name, founded by the original creators of Apache Project Name" or "several Company Name team members are the original developers of Apache Project Name."
```

Comment:
From a practical perspective, calling someone the/a "original developer of Apache Project Name" is accurate, but calling a company "founded by the original creators of Apache Project Name" can be misleading.

There can be more than one company whose employees are part of the original creators. And people may not stay in one company forever.

So, despite the individual's title, I am a bit more against this kind of company reference.

Reply:
Companies can state their product is based on "Apache Project Name", some of their employees are committers/PMC members, one or more certain certain persons originally developed the Apache Project Name before its entering the ASF (note that it now ties to individuals instead of directly to a company).

But stating the company is "founded by *the* original creators of Apache Project Name" trying to take an exclusive position from other vendors, which is even inaccurate.